packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { DropdownField } from '.';
+import { action } from '@storybook/addon-actions';
+import { text, boolean } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Primitives/Form/DropdownField',
+  component: DropdownField,
+};
+
+const options = [
+  { value: 1, label: 'Option 1' },
+  { value: 2, label: 'Option 2' },
+  { value: 3, label: 'Option 3' },
+];
+
```

Comment:
please add a story with the placeholder and another one with an item selected

Reply:
Done